### ISSUE TITLE
No TLE disables satellite rather than disabling position calcs for that satellite

### ISSUE THREAD

**Issue by burnsfisher** [ac2cz/FoxTelem#418]
What makes #417 especially bad is that if you check "FoxTelem calculates position" when there is no TLE for one of the tracked spacecraft, it gives a long message saying it is possible to track it but you must uncheck FoxTelem Calculates etc.  But then rather than unchecking FoxTelem calculates, it unchecks the satellite with no keps.

Ideally it would run for satellites with no keps just as it does without "FoxTelem calculates" while calculating for the rest.  2nd best would be unchecking "FoxTelem calculates" rather than unchecking the satellite itself.  My reasoning:  It's easy to get this wrong and not collect data from a satellite that you intend to.

**Comment by ac2cz** [comment]
I agree it could work better, but it does exactly what the error message says.  Let me see its its easy to just not calculate that sat....
![image](https://user-images.githubusercontent.com/14824793/34797786-763f1bc6-f651-11e7-9ce3-ec23a0269613.png)

**Comment by burnsfisher** [comment]
Small improvement would be to put the spacecraft name you are searching for
in the message.  When I first read "name of the spacecraft" it took me a
minute to realize you were talking about the name that was on the tab.  I
guess you could also say "the name as specified in the spacecraft menu" or
something.

Anyway, it's relatively minor.

**Comment by ac2cz** [comment]
I have done the obvious thing and just removed this warning.  It was to prevent people from wondering why it was not working, but I agree it probably does more harm than good.  Instead I will just display (No Position Info) or something like that next to the word "Tracked" in the top right.  There can be no position because the TLE or T0 data are missing.  I can explain in the manual.  In fact I think its in there from the last release.   

In 1.06g

**Comment by burnsfisher** [comment]
Fair enough!

**Comment by ac2cz** [comment]
Fixed in 1.06g
